CMakeLists.txt

```diff
@@ -248,6 +247,31 @@ configure_file(${CMAKE_SOURCE_DIR}/libraries/cli11/bash-completion/completions/c
 install(FILES libraries/cli11/bash-completion/completions/leap-util DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/bash-completion/completions COMPONENT base)
 install(FILES libraries/cli11/bash-completion/completions/cleos DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/bash-completion/completions COMPONENT base)
 
+set(TESTER_TARGETS fc secp256k1 bn256 chainbase builtins softfloat Logging IR WAST WASM Runtime eosio_chain eos-vm eosio_testlib)
+
+export(TARGETS ${TESTER_TARGETS}
+       FILE ${CMAKE_CURRENT_BINARY_DIR}/lib/cmake/leap/tester-targets.cmake)
+
+export(TARGETS ${TESTER_TARGETS}
+       FILE ${CMAKE_CURRENT_BINARY_DIR}/lib/cmake/eosio/tester-targets.cmake)
+
+set(SHIP_TARGETS ship_protocol abieos)
```

Comment:
It's not clear that leap should be a delivery mechanism for abieos. Actually my first thought is that it shouldn't be as it's not clear that we'll update abieos inside of leap at the same cadence as abieos repo itself. Needs more discussion.

Reply:
I had doubt for this as well. Kevin convinced me that this is what you guys had discussed and wanted. I am open to what solution you guys desires. Just let me know what you decided.